spec/requests/admin/articles_spec.rb

```diff
@@ -0,0 +1,138 @@
+require "rails_helper"
+
+RSpec.describe "Admin::Articles", type: :request do
```

Comment:
Hmm, I see that this is referring to `Admin::Articles` but I don't see this in the PR. Did you mean to have this run through the questions routes?

Reply:
I'm going to update the specs so that they are testing the Question model and controller next.

Reply:
I've updated this spec to test the question routes.